pkg/controller/registry/reconciler/configmap.go

```diff
@@ -288,6 +288,11 @@ func (c *ConfigMapRegistryReconciler) EnsureRegistryServer(logger *logrus.Entry,
 	overwrite := source.Status.RegistryServiceStatus == nil
 	overwritePod := overwrite
 
+	podSecurityConfig, err := getNamespaceSecurityContextConfig(c.OpClient, catalogSource.GetNamespace())
```

Comment:
WDYT about renaming this everywhere throughout the stack to `defaultPodSecurityConfig` to make it a bit more clear that this could be overridden if it is explicitly set in the catsrc?